README.md

```diff
@@ -273,6 +219,49 @@ The release configuration sets the following additional options:
 
 Logs can be found at http://localhost:9876/log/ode and SourceMaps at http://localhost:9876/sourcemaps/ode
 
+## Setup Instructions (Vagrant)
+
+Using Vagrant for App Inventor development is a method of last resort to get a development environment up and running is to use the provided Vagrantfile. Install [Vagrant](https://vagrantup.com) and open a terminal in the root directory of this repository. Run the following commands
```

Comment:
This sentence doesn't quite parse.  Perhaps you could just delete "is to use the provided Vagrantfile"?